my-app/.env

```diff
@@ -1 +1 @@
-REACT_APP_UNSPLASH_KEY=6ee32035bd5e430b923efffa01bdfc51
+REACT_APP_UNSPLASH_KEY=6e981d4adce646b6aeb2a2d500b56c6e
```

Comment:
**BAZINGA!!** Your `.env` file is in my bare hands, **Gosh** such a generous offer :blush:
 Now I can take your key and make as many requests as I want to get your site down, or I can be crueler and damage you more. you know, since the news are really boring it's extremely nice to have some fun playing around with your **EXPOSED SENSITIVE DATA** :slightly_smiling_face:

Reply:
🌚🌚 I will fix it